plugins/algoltek-usb/fu-algoltek-usb.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2023 Ling.Chen <ling.chen@algoltek.com.tw>
+// SPDX-License-Identifier: LGPL-2.1+
+
```

Comment:
I think you want your cmd enum and return code enum here instead.

Reply:
You mean moved the code in -common.h to here and nreamed the cmd name?

Reply:
Yeah the rust helper is good for enums.

Reply:
Updated, I move commands define to .rs file.